cfme/tests/physical_infrastructure/ui/test_physical_server_list_buttons.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+import pytest
+from selenium.common.exceptions import StaleElementReferenceException
+
+from cfme.physical.provider.lenovo import LenovoProvider
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.common.physical_server_views import (
```

Comment:
This import statement should be on one line, only use the parenthesis enclosed line-continuation when importing many classes/modules and they don't fit on 100 chars.